libcodechecker/server/server.py

```diff
@@ -628,9 +628,12 @@ def cleanup_run_db(self):
                                                       env=self.__check_env)
         with database.DBContext(prod_db) as db:
             try:
+                LOG.info("Garbage collection for product '%s' started...",
+                         self.endpoint)
                 db_cleanup.remove_expired_run_locks(db.session)
                 db_cleanup.remove_unused_files(db.session)
                 db.session.commit()
+                LOG.info("Garbage collection finished.")
                 return True
             except Exception as ex:
                 db.session.rollback()
```

Comment:
Shouldn't we log something here in case the garbage collection fails?

Reply:
We already write a message in case of failure:
https://github.com/Ericsson/codechecker/blob/114772b6f0a7da6a03fce4f25ffb7f74ea4d8511/libcodechecker/server/server.py#L637-L638